lib/diagnostic.js

```diff
@@ -46,10 +47,19 @@ let scopes = [/* your response here */];
 // method and temporarily bind it (or, change `this`) to `alderaan` before
 // executing it.
 //
+
+let result = deathStar.fire.call(alderaan);
 let fireOnAlderaan = function () {
-  // your code here
+  return result; // couldn't get it to run corectly inside of the function:(
```

Comment:
I think you may have mixed up `call` and `bind`. `call` and `apply` both bind the function to a new context AND execute the new function.

You could have written either:
```
let result = deathStar.fire.bind(alderaan);
let fireOnAlderaan = function(){
  return result();
};
```
or used `call`/`apply`.